### PR TITLE
Updated translation template (0000)

### PR DESCRIPTION
**Pull request by rmcpantoja**
Accessibility messages are added to this updated template.

### CODE REVIEW

File: Langs/0000.lang [hunk 1]
```diff
@@ -9,12 +9,15 @@ Medium=10
 Small=8.5
 Name=Arial
 [Strings]
+Enable enanced accessibility?=
```
Comment (micwoj92) [inline Langs/0000.lang:12]: enhanced

**Comment by micwoj92** [comment]
@rcmaehl This okay to be merged?

**Comment by mlipok** [comment]
> @rcmaehl This okay to be merged?

All other language files shoudl be fixed at the same PR

**Comment by micwoj92** [comment]
No, it's up to translators.

**Comment by krystian3w** [comment]
Or need found  cery good poligot than found 1-6 native speakers per language.

**Comment by micwoj92** [comment]
> Or need found very good poligot than found 1-6 native speakers per language.

Thank God you aren't "poligot"

**Comment by krystian3w** [comment]
I have papers for dyslexia, plus it's hard to spell foreign words from my phone as the dictionary suggests nardic.

**Comment by mlipok** [comment]
> Or need found very good polyglot than found 1-6 native speakers per language.



> No, it's up to translators.

Ah I see now.
I looked only on second commit not at first and not at all.
I thought that this is fixing typos, but first comment added this for first time.
So agree this is new feature and it is up to translators.

**Comment by mlipok** [comment]
So plealse change PR Title from "Updated....." to "Suplemented......" or "Enchanged......."
Or something like this.

**Comment by micwoj92** [comment]
@rmcpantoja please fix conflicts.

**Comment by rmcpantoja** [comment]
> @rmcpantoja please fix conflicts.

resolved

**Review by rcmaehl — APPROVED** [review]

**Pull request merged**